packages/core/src/components/image-overlay/image-overlay.scss

```diff
@@ -41,7 +43,7 @@ button {
   z-index: zindex.$image-overlay - 1;
   display: flex;
   padding: units.$u4;
-  background-color: fade-out(colors.$wit, 0.2);
+  background-color: color.scale(colors.$wit, $alpha: -20%);
```

Comment:
`-20%` is hetzelfde als `0.2`?

Reply:
Ja, het resulteert beide in `background-color: rgba(255, 255, 255, 0.8);`